engine/src/flutter/tools/licenses_cpp/src/license_checker.cc

```diff
@@ -26,8 +26,9 @@ namespace fs = std::filesystem;
 const char* LicenseChecker::kHeaderLicenseRegex = "(?i)(license|copyright)";
 
 namespace {
-const std::array<std::string_view, 5> kLicenseFileNames = {
-    "LICENSE", "LICENSE.TXT", "LICENSE.md", "LICENSE.MIT", "COPYING"};
+const std::array<std::string_view, 7> kLicenseFileNames = {
```

Comment:
This was a gotcha.  macOS is usually built on top of a case-insensitive filesystem, where linux is not.